gusto/linear_solvers.py

```diff
@@ -136,14 +139,18 @@ def _setup_solver(self):
         def V(u):
             return k*inner(u,k)
 
+        # specify degree for some terms as estimated degree is too large
+        dxp = dx(degree=(5,5))
```

Comment:
the point is more that if you change gusto in the future to support higher than RT1 (say), this integration degree will be wrong.  Can you determine the value to put in programmatically by asking for the degree of the function spaces in the pi term?